jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: classic-docs
+title: "Insights metrics glossary"
+short-title: "Insights metrics glossary"
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
+
+This document provides definitions for all the metrics in CircleCI Insights. You
+can review metrics in the Insights dashboard as well as the Insights API.
+
+## General metrics 
+
+General metrics appear across the Insights experience and can refer to different
+entities, depending on the context. For example, the `Runs` metric may refer to a
+count of workflow executions or jobs, depending on the context.
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Runs                    | The count of executions in a selected time window. |
+| Total Duration          | The sum of execution times for all workflows in a project. |
+| P95 Duration            | The 95th percentile of execution times for an entity in a selected time window (i.e. 95% of runs completed in the same or less amount of this time). _The 95th percentile is a standard measure used to interpret performance data. It provides a measure of max value when outlier or transient values are excluded._ |
+| P50 (median) Duration   | The median execution time. _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_. |
+| Total Credits           | The sum of credits consumed during execution. |
+| Success Rate            | The percentage of runs that completed successfully, calculated by 100 x (Successful Runs / All Runs) |
+{: class="table table-striped"}
+
+## Organization-level metrics
+
+Organization-level metrics allow you to analyze your organization’s performance.
+
```

Comment:
Can we add an image here to show what these metrics will look like? There's an image in the google doc. Happy to provide a higher fidelity one if needed too